packages/preview/diagraph/0.2.1/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2023 Robotechnic
```

Comment:
I don't know, how it's usually done for licenses, but maybe that's good idea to change year to 2024